pkg/profiler/profiler.go

```diff
@@ -53,6 +54,16 @@ const (
 	doubleStackDepth = 254
 )
 
+// stackCountKey mirrors the struct in parca-agent.bpf.c.
+//
+// TODO(derekparker) Perhaps in order to keep these in sync we should write a Go generator to
```

Comment:
Awesome idea! Let's do it.

Reply:
Already the opened the issue for it https://github.com/parca-dev/parca-agent/issues/207